servers.json

```diff
@@ -73,5 +73,8 @@
   },
   {
     "address": "Chaotic-Neutral.ddns.net:3333"
+  },
+  {
+    "address": "Zap-hosting.com :88.214.59.19:36200"
```

Comment:
I'm unable to connect to `Zap-hosting.com :88.214.59.19:36200` nor `Zap-hosting.com`
`88.214.59.19:36200` works fine. Also, could you provide the required mods for this server?

Reply:
im trying to get it put on the community list Anuke wont let me use mods if i want it there i guess i failed to get it to standards again